src/NuGetPackageVerifier/CompositeRules/PackageDependenciesCompositeRule.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class PackageDependenciesCompositeRule : IPackageVerifierRule
```

Comment:
So I found [this](https://github.com/aspnet/Coherence-Signed/blob/dev/tools/NugetPackageVerifier.json). Since it isn't running the `DefaultCompositeRule`, for now I am adding the new composite rule with just the pre-rel package rule to the list at the end of this file.